cometd-java/cometd-java-server/cometd-java-server-common/src/main/java/org/cometd/server/BayeuxServerImpl.java

```diff
@@ -1259,23 +1266,25 @@ private boolean validateSubscriptions(List<String> subscriptions) {
 
     @ManagedOperation(value = "Sweeps channels and sessions of this CometD server", impact = "ACTION")
     public void sweep() {
-        sweepChannels();
+        _channels.values().forEach(ServerChannelImpl::sweep);
         sweepTransports();
-        sweepSessions();
+        long now = System.nanoTime();
+        for (ServerSessionImpl session : _sessions.values()) {
+            session.sweep(now);
+        }
```

Comment:
Let change the order to be: transports, channels, sessions.

Reply:
Done.